tests/utils/config/support.go

```diff
@@ -73,7 +76,27 @@ func DeployCilium(ocClient *occli.Client, podCIDR string, hostPrefix string, out
 	return err
 }
 
-func GetKubeconfigDummyFunc() {
+func GetKubeconfigDummyFunc(idpService rosacli.IDPService, userService rosacli.UserService, clusterID string) error {
```

Comment:
- Change the function name

Reply:
Function can not be placed in config package. Importing module "github.com/openshift/rosa/tests/utils/exec/rosacli" is making an error "Import cycle not allowed". My suggestion are to use the code directly without function or create a new module to include the function. @yingzhanredhat what is your suggestion to solve this error ?